imagick/imagick.go

```diff
@@ -251,9 +254,15 @@ func (i *Image) sizeFrames(sz *imgry.Sizing) error {
 		i.mw = i.mw.CoalesceImages()
 	}
 
+	if sz.Canvas != nil {
+		// If the user requested a canvas.
+		canvas = imagick.NewMagickWand()
+		bg = imagick.NewPixelWand()
+		bg.SetColor("transparent")
```

Comment:
Can you make sure that you are destroying/releasing both the canvas (MagickWand) and bg (PixelWand) once you're done with it...? imagick is notorious for leaking memory